### ISSUE TITLE
Not able to install tomorrow-theme

### ISSUE THREAD

**Issue by FriendOfFatBeagle** [chriskempson/tomorrow-theme#302]
Hello Chris:

Linux Mint 17, 64-bits.
I installed your theme in vim via Vundle as shown:

" Installing plugins to /home/foffb/.vim/bundle
Plugin 'gmarik/Vundle.vim'
Plugin 'tpope/vim-fugitive'
Plugin 'rstacruz/sparkup'
Plugin 'chriskempson/tomorrow-theme'
Helptags
============================
Then I set it up variables like this:

syntax enable
set t_Co=256
set background=light
"??For dark mode, use set background=dark instead of light
colorscheme Tomorrow
=========================
And I got error like this:
$ gvim vimrc
Error detected while processing /home/user1/.vim/vimrc:
line   66:
E185: Cannot find color scheme 'Tomorrow'
=============
I notice many subfolders before reaching Tomorrow.vim
/home/foffb/.vim/bundle/tomorrow-theme/vim/colors/Tomorrow.vim

What did I missed in my Vundle's vimrc??
Thanks.

**Comment by markbao** [comment]
For those who are searching for this in the future, you need to specify which folder Vundle should find the plugin in, like so:

```
Plugin 'chriskempson/tomorrow-theme', {'rtp': 'vim/'}
```

**Comment by bearcatsandor** [comment]
Would the setup be the same for vim-plug?

**Comment by TylerBrock** [comment]
Yeah it worked for me @bearcatsandor `Plug 'chriskempson/tomorrow-theme', { 'rtp': 'vim' }`
